dev/PushNotifications/PushNotificationUtility.h

```diff
@@ -141,7 +141,19 @@ namespace winrt::Microsoft::Windows::PushNotifications::Helpers
 
     inline wil::com_ptr<INotificationsLongRunningPlatform> GetNotificationPlatform()
     {
-        return wil::CoCreateInstance<INotificationsLongRunningPlatform>(CLSID_NotificationsLongRunningPlatform, (CLSCTX_LOCAL_SERVER | CLSCTX_ALLOW_LOWER_TRUST_REGISTRATION));
+        // If supported, allow use of class registrations from less trusted sources.
+        auto supportMarker{ wil::CoCreateInstanceNoThrow<ISupportAllowLowerTrustActivation>(CLSID_ActivationCapabilities) };
```

Comment:
Probably best to cache supportMarker; it shouldn't change within the app's lifetime.

Reply:
This will be wrapped up std::call_once

Reply:
Fixed.